client/landing/gutenboarding/components/domain-categories/style.scss

```diff
@@ -1,5 +1,47 @@
 @import '../../mixins';
 
+.domain-categories {
+
+	@media ( max-width: $break-medium ) {
```

Comment:
Would there be similar mixin to `@include break-mobile` for medium size?

Reply:
We could  introduce a new mixin and I'm not sure what to call that. `@include break-mobile` uses `min-width` instead of `max-width`.

Reply:
Oh right, didn't realize that `break-medium()` et co are all `min-width` 👍